util/os-runner/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN apt-get update &&   \
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-COPY install-dmd.sh /init-scripts/install-dmd.sh
+COPY util/os-runner/install-dmd.sh /init-scripts/install-dmd.sh
```

Comment:
Will the current `Makefile` work with this change?

Reply:
Ignore for the moment, I'm doing some tests